modules/apps/frontend-theme/kyoung-liferay-theme/src/WEB-INF/src/resources-importer/journal/templates/Welcome-section/Welcome-section.ftl

```diff
@@ -0,0 +1,26 @@
+<div class="header-featured-area">
+    <div class="container">
+        <div class="header-featured-img">
+            <#if Imagee3k2.getData()?? && Imagee3k2.getData() !="">
+				<img alt="${Imagee3k2.getAttribute("alt")}" data-fileentryid="${Imagee3k2.getAttribute("fileEntryId")}" src="${Imagee3k2.getData()}" />
+			</#if>
+        </div>
+        <div class="header-featured-content">
+            <h1>${WelcomeSummaryTitle.getData()}</h1>
+            <p>${WelcomeSummary.getData()}</p>
+            <div class="app-download-area">
+                <h4>
+					<@liferay.language key="download-the-app" />

```

Comment:
Indentations should be 1 tab/4 spaces.  In Liferay, we use 1 tab as our standard indentation.